tfhe/benches/shortint/bench.rs

```diff
@@ -304,6 +306,39 @@ fn programmable_bootstrapping(c: &mut Criterion) {
     bench_group.finish();
 }
 
+fn server_key_from_compressed_key(c: &mut Criterion) {
+    let mut bench_group = c.benchmark_group("uncompress_key");
+    bench_group
+        .sample_size(10)
+        .measurement_time(std::time::Duration::from_secs(60));
+
+    for param in SERVER_KEY_BENCH_PARAMS {
```

Comment:
no multi bit ? don't remember how we manage benchmarks params

Reply:
Thing is we don't have any multi-bit parameters in shortint. Should I move this benchmark to integer layer then ?

Reply:
There should be multi bits params in shortint, check parameters::multi_bit or similar